src/styles/LandingPage.module.less

```diff
@@ -71,6 +75,13 @@
     background-color: #3fcafc; 
 }
 
+.LabDescription{
+display: inline-block;
+font-size: 1.375rem;
+line-height: 54px;
+width:25vw;
```

Comment:
indentations, we don't have any eslint for css & less so these have to be manual